control/init.sh

```diff
@@ -22,14 +23,19 @@ if [ -z ${CI_TEST} ]; then
     if [[ $BIONIC_USE_SYSTEM_PYTHON == false ]] && hash python3.10 2>/dev/null && [[ $( source /etc/os-release; echo $VERSION_ID ) == 18.04 ]]; then
         # only append 3.10 if it is not already in the name
         if [[ $CCHQ_VIRTUALENV != *"3.10"* ]]; then
-          CCHQ_VIRTUALENV=$CCHQ_VIRTUALENV-3.10
+            OLD_VENV=$VENV
```

Comment:
I'm not too strongly attached to any naming convention here, but seeing as we set the path of the current virtualenv to `VENV`, I don't see why `OLD_VENV` isn't good enough. (An explanation along with the suggestion would be helpful)

Reply:
I don't have a strong opinion here either. Since I had other suggestions it made sense to me to change the variable to something that would be more specific to us, avoiding possible conflicts in the user env.

Reply:
Cool. I updated the var in 895c590c8591c01c0b1c5595059894944ce5d1c9 to `CCHQ_VENV_PATH_OLD`.